src/testCase.js

```diff
@@ -88,6 +88,8 @@ const newTestCase = (caseIndex) => {
   }
 }
 
+// Not sure what `args` is for here, though the lint warning seems correct.
+// eslint-disable-next-line no-unused-vars
 const testCaseFn = (actionArgs, action, paramNames, args = []) => {
```

Comment:
This is the error:

```
/Users/pnm/code/sazerac/src/testCase.js
  91:53  error  'args' is assigned a value but never used  no-unused-vars
```

I can't figure out what `args` is for here. Should it be removed? Not sure, so I left a comment instead.